### PR TITLE
fix-unittest-"Feature/start-es-k8s-(#626)"

### PR DESCRIPTION
**Pull request by drsantos89**
Fixes #{issue-id-number}.

## Description

Please provide here a summary of the changes introduced by this PR.

## How to test?

Please provide here instructions on how to test the changes introduced by this PR.
(if some changes cannot be tested by automated tests)

## Checklist

- [ ] This PR refers to an issue from the [issue tracker](https://github.com/BlueBrain/Search/issues).
  (if it is not the case, please create an issue first).
- [ ] Unit tests added.
  (if needed)
- [ ] Documentation and `whatsnew.rst` updated.
  (if needed)
- [ ] `setup.py` and `requirements.txt` updated with new dependencies.
  (if needed)
- [ ] Type annotations added.
  (if a function is added or modified)
- [ ] All CI tests pass.

### CODE REVIEW

File: src/bluesearch/k8s/create_indices.py [hunk 1]
```diff
@@ -47,12 +64,12 @@
...
 
 def add_index(
+    client: Elasticsearch,
     index: str,
     settings: dict[str, Any] | None = None,
     mappings: dict[str, Any] | None = None,
 ) -> None:
-    client = connect()
-
+    """Add the index to ES."""
```
Comment (EmilieDel) [inline src/bluesearch/k8s/create_indices.py:72]: Can we add the description of the parameters and the returned values ?

File: src/bluesearch/k8s/create_indices.py [hunk 2]
```diff
@@ -63,9 +80,8 @@ def add_index(
         print("Elasticsearch add_index ERROR:", err)
 
 
-def remove_index(index: str | list[str]) -> None:
-    client = connect()
-
+def remove_index(client: Elasticsearch, index: str | list[str]) -> None:
+    """Remove the index from ES."""
```
Comment (EmilieDel) [inline src/bluesearch/k8s/create_indices.py:84]: Same comment here.

**Review by EmilieDel — APPROVED** [review]
LGTM! Thank you for taking care of the CI!

File: src/bluesearch/k8s/create_indices.py (hunk 1 above)
Comment (drsantos89) [reply src/bluesearch/k8s/create_indices.py]: done

File: src/bluesearch/k8s/create_indices.py (hunk 2 above)
Comment (drsantos89) [reply src/bluesearch/k8s/create_indices.py]: done

**Pull request merged**